backend/app/controllers/spree/admin/stock_items_controller.rb

```diff
@@ -50,7 +50,11 @@ def load_stock_management_data
           @stock_item_stock_locations = params[:stock_location_id].present? ? @stock_locations.where(id: params[:stock_location_id]) : @stock_locations
           @variant_display_attributes = self.class.variant_display_attributes
           @variants = Spree::Config.variant_search_class.new(params[:variant_search_term], scope: variant_scope).results
-          @variants = @variants.includes(:images, stock_items: :stock_location, product: :variant_images)
+          @variants = @variants.includes(
+                                         Spree::Config.variant_gallery_class.preload_params,
```

Comment:
I'd prefer some simpler indentation rather than sticking everything far to the right:

```Ruby
@variants = @variants.includes(
  Spree::Config.variant_gallery_class.preload_params,
  stock_items: :stock_location,
  product: Spree::Config.product_gallery_class.preload_params
)
```